tests/module-for-model-test.js

```diff
@@ -3,16 +3,18 @@ import DS from 'ember-data';
 import { moduleForModel, test } from 'ember-qunit';
 import { setResolverRegistry } from 'tests/test-support/resolver';
 
+var Adapter = DS.JSONAPIAdapter || DS.FixutreAdapter;
```

Comment:
@Turbo87 - 😭  I got so caught up in deleting that other test.  Should have waited for next round of review, thank you for catching it (again).